scripts/src/chartprreview/chartprreview.py

```diff
@@ -274,20 +297,30 @@ def generate_verify_report(directory, category, organization, chart, version):
             sys.exit(1)
     kubeconfig = os.environ.get("KUBECONFIG")
     if not kubeconfig:
-        msg = "[ERROR] missing 'KUBECONFIG' environment variabl"
+        msg = "[ERROR] missing 'KUBECONFIG' environment variable"
+        write_error_log(directory, msg)
+        sys.exit(1)
+    vendor_type = os.environ.get("VENDOR_TYPE")
+    if not vendor_type:
+        msg = "[ERROR] missing 'VENDOR_TYPE' environment variable"
```

Comment:
Maybe another message since for the user perspective this means they did not place the chart or report in the right directory.

Reply:
updated